swift/Samples/SplitScreenContainer/Sources/SplitScreenContainerViewController.swift

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Workflow
+import WorkflowUI
+
+public final class SplitScreenContainerViewController<LeftScreenType: Screen, RightScreenType: Screen>: ScreenViewController<SplitScreenContainerViewController.ContainerScreen> {
+    public typealias ContainerScreen = SplitScreenContainerScreen<LeftScreenType, RightScreenType>
```

Comment:
Should this be `leading` and `trailing` instead of `left/right`?

Reply:
Resolved via https://github.com/square/workflow/pull/925/commits/adcea08dffb1c21a3a00f903e77f1ef8b1c299e7